src/lib/containers/entity_finder/EntityFinderModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import { Reference } from '../../utils/synapseTypes'
+import { FluidModal } from '../FluidModal'
+import EntityFinder, { EntityFinderProps } from './EntityFinder'
+
+type EntityFinderModalProps = {
+  configuration: Omit<EntityFinderProps, 'onSelectedChange'>
+  show: boolean
+  onClose: () => void
+  title: string
+  onConfirm: (selected: Reference[]) => void
+  confirmButtonCopy: string
+  onCancel: () => void
+}
+
+export const EntityFinderModal = (props: EntityFinderModalProps) => {
```

Comment:
Similar to SWC, we'll have a reusable encapsulation of the EntityFinder in a modal.